pylint/utils.py

```diff
@@ -1157,6 +1158,12 @@ def _ini_format(stream, options, encoding):
             print('#%s=' % optname, file=stream)
         else:
             value = _encode(value, encoding).strip()
+            if section == 'MESSAGES CONTROL' and ',' in str(value):
+                separator = '\n ' + ' ' * len(optname)
+                value = separator.join(
```

Comment:
Super small nit - there's no need for the list comprehension here.  

```python
value = separator.join(x + ',' for x in str(value).split(','))
```

Will work just fine.  See https://www.python.org/dev/peps/pep-0289/

Now to figure out how to write a checker for this  😆